File: python/leccion3/ejercicio1.py
mes = int(input("Ingrese un mes del año (1-12): "))

if mes == 12 or mes == 1 or mes == 2:
    print("Usted se encuantra en Verano")
elif mes == 3 or mes == 4 or mes == 5:
    print("Usted se encuantra en Otoño")
elif mes == 6 or mes == 7 or mes == 8:
    print("Usted se encuentra en Invierno")
elif mes == 9 or mes == 10 or mes == 11:
    print("Usted se encuentra en Primavera")
else:
    print(f"Usted ha ingresado {mes}: NO ES VÁLIDO")

File: java/CondicionalEjercicio_1/src/main/java/Ejercicio1/Ejercicio1.java
package Ejercicio1;
import java.util.Scanner;

public class Ejercicio1 {
    public static void main(String[] args) {
        Scanner entrada = new Scanner(System.in);
        System.out.println("Digite un mes del año: ");
        var mes = Integer.parseInt(entrada.nextLine());
        var estacion = "Estación desconocida.";
        if (mes == 12 || mes == 1 || mes == 2) {
            estacion = "Usted se encuentra en verano.";
        } else if (mes == 3 || mes == 4 || mes == 5) {
            estacion = "Usted se encuentra en otoño";
        } else if (mes == 6 || mes == 7 || mes == 8) {
            estacion = "Usted se encuentra en invierno.";
        } else if (mes == 9 || mes == 10 || mes == 11)
            estacion = "Usted se encuentra en primavera.";
        System.out.println(estacion);
    }
}
